student-bios/Renjani2022.md

```diff
@@ -0,0 +1,10 @@
+## Introduction
+Hi, I'm Renjani!
```

Comment:
you have a format error: 

> Checking formatting...
[warn] student-bios/Renjani2022.md
[warn] Code style issues found in the above file. Forgot to run Prettier?
Error: Process completed with exit code 1.

Please run the scripts in the terminal one by one 
npm run format
npm run format:check
npm run lint:ls
npm run lint:md
npm run spell-check

Reply:
Thanks @yildiztugba. I have done now. And what shall I do next? Shall I apply for the new pull request? Thanks a lot

Reply:
i made 2 comments, please look, you have still format error. dont forget to run script and push again